test/e2e-tests/README.md

```diff
@@ -16,17 +16,33 @@ The following enviroment varaibles are required to run the test setup and the te
 - `S3_ENDPOINT` - Endpint of the bucket
 - `IMAGE_REGISTRY_USERNAME` - quay.io username
 - `IMAGE_REGISTRY_PASSWORD` - quay.io password
-- `S3_BUCKET` - Name of S3 bucket that has the model
-- `TARGET_IMAGE_TAGS_JSON` - JSON array of image tags that the final image will be pushed to. E.g. '["quay.io/user/model-name:e2e-test"]'
-- `NAMESPACE` - Cluster namespace used for testing
+
+The following enviroment varaibles are optional. You may still need to set them for the tests to pass depending on your setup. Read [here](../../pipelines/README.md#ai-edge-end-to-end-pipeline) for more context on how to set these up.
+- `SELF_SIGNED_CERT` - Path self signed cert to be used in pipeline
```

Comment:
Can you add a small clarification that this is the local path on the  user's machine where the self-signed cert file is located

Reply:
👍